ale/drivers/mro_drivers.py

```diff
@@ -39,27 +42,25 @@ def spacecraft_id(self):
         : int
           Naif ID code for the spacecraft
         """
-        return "N/A"
+        return "-74"
 
     @property
-    def ikid(self):
+    def sensor_model_version(self):
         """
         Returns
         -------
         : int
-          Naif ID code for the instrument
+          ISIS sensor model version
         """
-        return int(self.label["IsisCube"]["Kernels"]["NaifFrameCode"])
+        return 1
```

Comment:
I'm not sure how this is used by CSM, so perhaps it doesn't matter, but couldn't this come from ["IsisCube"]["Kernels"]["CameraVersion"] ?

Reply:
This is not used at all by CSM, but this value should be coming in off of that location. That code probably needs to be added to data_isis